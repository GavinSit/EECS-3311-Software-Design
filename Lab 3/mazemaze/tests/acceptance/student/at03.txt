--try solving
new_game(easy)
solve
new_game(medium)
solve
new_game(hard)
