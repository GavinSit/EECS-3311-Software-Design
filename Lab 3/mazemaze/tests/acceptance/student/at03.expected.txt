  State: 0 -> ok
->new_game(easy)
  State: 1 -> ok
  +-*-+---+---+---+---+---+---+
  | X                         |
  +   +   +   +---+---+   +   +
  |       |   |       |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->solve
  State: 2 -> ok
  +-*-+---+---+---+---+---+---+
  | X   o   o                 |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->new_game(medium)
  State: 3 -> Error! In game already.
  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->solve
  State: 4 -> ok
  +-*-+---+---+---+---+---+---+
  | X   o   o                 |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->new_game(hard)
  State: 5 -> Error! In game already.
  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

